pkgs/development/mobile/checkra1n/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "checkra1n";
+  version = "0.12.4";
+
+  src = fetchurl {
+    url = "https://assets.checkra.in/downloads/linux/cli/x86_64/dac9968939ea6e6bfbdedeb41d7e2579c4711dc2c5083f91dced66ca397dc51d/checkra1n";
```

Comment:
I don't like that there is no versioned download or update script.

Reply:
I'll see if they provide versioned downloads, but I'm not sure

Reply:
Looks like the checksum represents the specific version / release. They don't have any other binary distribution :(